js/src/components/paid-ads/asset-group/asset-group-card.js

```diff
@@ -126,8 +126,8 @@ export default function AssetGroupCard() {
 					</AssetField>
 				);
 			} ) }
-			{ ASSET_TEXT_SPECS.map( ( spec, index ) => {
-				const initialTexts = baseAssetGroup[ spec.key ];
+			{ ASSET_TEXT_SPECS.map( ( spec ) => {
+				const initialTexts = [ baseAssetGroup[ spec.key ] ].flat();
```

Comment:
As far as I know this would be also impacted by #1907

Reply:
A code conflict has been resolved in https://github.com/woocommerce/google-listings-and-ads/pull/1908/commits/1759264a5e55e5763850dca8c051bb7c516b9a2b.